templates/comments/comments_tree.html

```diff
@@ -42,7 +42,7 @@
                 | <a href="{% url 'comment_edit' comment_id=comment.id %}">edit</a>
                 | <a href="{% url 'comment_delete' comment_id=comment.id %}">delete</a>
                 {% endifequal %}
-                <a class="togg" n="{{ comment.comment_set.count|add:" 1" }}"
```

Comment:
This gives only the number of immediate children, and not the total number of descendants of the node. Updated it to use code that determines which comments to hide (comments that have to be hidden = all descendants of the node).

Reply:
Hi Ravi, why did you change the div element to the img one with s.gif? Is not more elegant to use div with style instead? Just asking, we are not frontend experts :D

Reply:
With the div, the width is inside the `style` property. It is slightly more verbose to fetch this value when calculating which comments to collapse. With an `img` the width property is directly accessible. :)